src/sdp.ts

```diff
@@ -62,16 +62,16 @@ function certhashToFingerprint(ma: Multiaddr): string {
   }
 
   let fp = mhdecoded.digest.reduce((str, byte) => str + byte.toString(16).padStart(2, '0'), '');
```

Comment:
```suggestion
  const fp = mhdecoded.digest.reduce((str, byte) => str + byte.toString(16).padStart(2, '0'), '');
```

again this shows me that our linting is pretty relaxed, as I would expect the linter to complain about this kind of stuff?